src/languages/clickhouse/clickhouse.keywords.ts

```diff
@@ -0,0 +1,782 @@
+  'MINUTE',
+  'MINUTES',
+  'MM',
+  'MOD',
+  'MODIFY',
+  'MONTH',
+  'MONTHS',
+  'MOVE',
+  'MS',
+  'MUTATION',
+  'N',
+  'NAME',
+  'NAMED',
+  'NANOSECOND',
+  'NANOSECONDS',
+  'NEXT',
+  'NO',
+  'NONE',
+  'NOT',
+  'NO_PASSWORD',
+  'NS',
+  'NULL',
+  'NULLS',
+  'OBJECT',
+  'OFFSET',
+  'ON',
+  'ONLY',
+  'OPTIMIZE',
+  'OPTION',
+  'OR',
+  'ORDER',
+  'OUTER',
+  'OUTFILE',
+  'OVER',
+  'OVERRIDABLE',
+  'OVERRIDE',
+  'PART',
+  'PARTIAL',
+  'PARTITION',
+  'PARTITIONS',
+  'PART_MOVE_TO_SHARD',
+  'PASTE',
+  'PERIODIC',
+  'PERMANENTLY',
+  'PERMISSIVE',
+  'PERSISTENT',
+  'PIPELINE',
+  'PLAINTEXT_PASSWORD',
+  'PLAN',
+  'POLICY',
+  'POPULATE',
+  'PRECEDING',
+  'PRECISION',
+  'PREWHERE',
+  'PRIMARY',
+  'PRIVILEGES',
+  'PROCESSLIST',
+  'PROFILE',
+  'PROJECTION',
+  'PROTOBUF',
+  'PULL',
+  'Q',
+  'QQ',
+  'QUALIFY',
+  'QUARTER',
+  'QUARTERS',
+  'QUERY',
+  'QUOTA',
+  'RANDOMIZE',
+  'RANDOMIZED',
+  'RANGE',
+  'READONLY',
+  'REALM',
+  'RECOMPRESS',
+  'RECURSIVE',
+  'REFERENCES',
+  'REFRESH',
+  'REGEXP',
+  'REMOVE',
+  'RENAME',
+  'REPLACE',
+  'RESET',
+  'RESPECT',
+  'RESTORE',
+  'RESTRICT',
+  'RESTRICTIVE',
+  'RESUME',
+  'REVOKE',
+  'RIGHT',
+  'ROLE',
+  'ROLES',
+  'ROLLBACK',
+  'ROLLUP',
+  'ROW',
+  'ROWS',
+  'S',
+  'S3',
+  'SALT',
+  'SAMPLE',
+  'SAN',
+  'SCHEME',
+  'SECOND',
+  'SECONDS',
+  'SECURITY',
+  'SELECT',
+  'SEMI',
+  'SEQUENTIAL',
+  'SERVER',
+  'SET',
+  'SETS',
+  'SETTING',
+  'SETTINGS',
+  'SHA256_HASH',
+  'SHA256_PASSWORD',
+  'SHARD',
+  'SHOW',
+  'SIGNED',
+  'SIMPLE',
+  'SNAPSHOT',
+  'SOURCE',
+  'SPATIAL',
+  'SQL',
+  'SQL_TSI_DAY',
+  'SQL_TSI_HOUR',
+  'SQL_TSI_MICROSECOND',
+  'SQL_TSI_MILLISECOND',
+  'SQL_TSI_MINUTE',
+  'SQL_TSI_MONTH',
+  'SQL_TSI_NANOSECOND',
+  'SQL_TSI_QUARTER',
+  'SQL_TSI_SECOND',
+  'SQL_TSI_WEEK',
+  'SQL_TSI_YEAR',
+  'SS',
+  'SSH_KEY',
+  'SSL_CERTIFICATE',
+  'STALENESS',
+  'START',
+  'STATISTICS',
+  'STDOUT',
+  'STEP',
+  'STORAGE',
+  'STRICT',
+  'STRICTLY_ASCENDING',
+  'SUBPARTITION',
+  'SUBPARTITIONS',
+  'SUSPEND',
+  'SYNC',
+  'SYNTAX',
+  'SYSTEM',
+  'TABLE',
+  'TABLES',
+  'TAGS',
+  'TEMPORARY',
+  'TEST',
+  'THAN',
+  'THEN',
+  'TIES',
+  'TIME',
+  'TIMESTAMP',
+  'TO',
+  'TOP',
+  'TOTALS',
+  'TRACKING',
+  'TRAILING',
+  'TRANSACTION',
+  'TREE',
+  'TRIGGER',
+  'TRUE',
+  'TRUNCATE',
+  'TTL',
+  'TYPE',
+  'TYPEOF',
+  'UNBOUNDED',
+  'UNDROP',
+  'UNFREEZE',
+  'UNION',
+  'UNIQUE',
+  'UNSET',
+  'UNSIGNED',
+  'UNTIL',
+  'UPDATE',
+  'URL',
+  'USE',
+  'USER',
+  'USING',
+  'UUID',
+  'VALID',
+  'VALUES',
+  'VARYING',
+  'VIEW',
+  'VISIBLE',
+  'VOLUME',
+  'WATCH',
+  'WATERMARK',
+  'WEEK',
+  'WEEKS',
+  'WHEN',
+  'WHERE',
+  'WINDOW',
+  'WITH',
+  'WITH_ITEMINDEX',
+  'WK',
+  'WRITABLE',
+  'WW',
+  'YEAR',
+  'YEARS',
+  'YY',
+  'YYYY',
+  'ZKPATH',
+];
+
+export const keywordPhrases: string[] = [
+  // See documentation of `keywords` above
+  'ADD COLUMN',
```

Comment:
These should really be just part of keywords.

The -phrases variables should be used for the cases where you're using the `expandPhrases()` function to expand expressions like `TIMESTAMP {WITH | WITHOUT} TIME ZONE`.

When you just need multi-word keywords, they work just fine out of the box. Just like you're using multi-word data type names further down.